src/components/Articles.jsx

```diff
@@ -1,5 +1,25 @@
-function Articles(){
-    
+import { useEffect, useState } from 'react';
+import allArticles from '../../api';
+import ArticleCard from './ArticleCard';
+
+function Articles() {
+	const [articles, setArticles] = useState([]);
+
```

Comment:
Please include a loading pattern to improve the user experience while they’re waiting for the page to render.